### DISCUSSION TITLE
Artist Creator - Seeking targeted audiences within an hour's drive of Toronto to attend my Multidisciplinary Theatre Show.

### DISCUSSION THREAD

**Discussion by ghost** [pair-cg/general-contributions#89] [User Stories]
# 1. Motivating Story
I'm the creator of an **interdisciplinary theatrical performance** made under the banner of BetterHalf Productions, an Independent art studio based in Scarborough, Toronto, ON. This show runs for 16 days in February 2022 and consists of live theatre, video projection, and acoustic music. I am seeking audiences in four demographic groups within an hour's drive of Toronto GTA to attend my Multidisciplinary Theatre Show at Young Centre for the Performing Arts, Toronto. We already have pre-planned marketing for other remaining cultural groups, who live within Toronto GTA. In a recent survey organized by Scarborough Arts (one of Toronto’s six Local Arts Service Organizations, we found that these four demographic groups are least likely to attend any art performances. Three out of five people from these demographic groups have never attended any theatre show in the last five years.

# 2. Need:
I need _South Asian, Asian, Caribbean, and Asian-Pacific Islander_ audiences to attend this show. I am telling the story based on the collective history of this group, and I want them to see themselves in this presentation.  

Here are the competency question based on my need. 

CQ: What is the name of the production?
Answer: The Land you promised.
Proposed terminology: title
Instance of title: The Land you promised

CQ: What style of production is it?
Answer: It's an interdisciplinary theatrical performance
Proposed terminology: production_type
Instance of production_type: interdisciplinary theatrical performance.

CQ: What demographic groups I am targeting? 
Answer: I am targeting South Asian, Asian, Caribbean, and Asian-Pacific Islander communities.
Proposed terminology: Demographic
Instance of demographic: South Asian, Asian, Caribbean, and Asian-Pacific Islander 

CQ: Why we are targeting given demographics in promotional activities? 
Answer: The production is based on the collective history of mentioned communities and it's an opportunity to relive shared experiences on stage via this performance. 
Proposed terminology: reason_for_target_demographics 
Instance of reason_for_target_demographics: collective history, shared experiences. 

CQ: What are we offering to targeted audiences living outside Toronto? 
Answer: We are offering 10% Discount on ticket purchases.
Proposed terminology: offers_targeted_audiences
Instance of offers_targeted_audiences: 10% off on tickets

CQ: What are we offering to those who will book multiple tickets?
Answer: We are offering group discounts and family passes. 
Proposed terminology: offers_multiple_tickets
Instance of offers_multiple_tickets: group discounts and family pass.

CQ: what is the criteria to avail group discount?
Answer: Anyone can avail group discount if they are booking more than 15 tickets.
Proposed terminology: group_discount_criteria 
Instance of group_discount_criteria: more than 15 tickets.

CQ: What's the percentage of group discount? 
Answer: group discount consist of 15% off up to 15 tickets and 20% off on more than 15 tickets.  
Proposed terminology: percentage_group_discount
Instance of percentage_group_discount: 15% - 20% 

CQ: What is the criteria to get family pass?
Answer: A family pass is available for any household purchasing more than 4 tickets. 
Proposed terminology: family_pass_criteria
Instance of family_pass_criteria: more than 4 tickets.

CQ: What are the components of Family Pass?
Answer: Family pass consist of discounted tickets, a interactive booklet on the background of performance, and a private invitation to attend a virtual reception after the production ends. 
Proposed terminology: family_pass_components
Instance of family_pass_components: discounted tickets, interactive booklet and a private invitation to virtual reception. 

CQ: How can the targeted audience can access these offers? 
Answer: They can access these offers by using a promo code. 
Proposed terminology: how_to_avail_offer
Instance of how_to_avail_offer: By using promo Code. 

CQ: How  targeted audience can receive promo code?
Answer: targeted audience can receive promo code by emailing at Offers@bhroductions.ca 
Proposed terminology: promo_code_access 
Instance of promo_code_access: Email us at Offers@bhroductions.ca 

CQ: Which social media platforms are we targeting for location based social media campaigns? 
Answer: We are targeting Facebook and Instagram
Proposed terminology: targeted_social_media_plateforms
Instance of targeted_social_media_plateforms: Facebook and Instagram

CQ: Which features of Facebook are we using for our outreaching campaign? 
Answer: We are using Facebook Ads
Proposed terminology:  Facebook_feature_to_campaign 
Instance of Facebook_feature_to_campaign : Facebook Ads 

CQ: Which feature of Instagram are we using for our outreaching campaign?
Answer: We have te-up with Instagram influencers to spread the word regarding the performance run. 
Proposed terminology: Instagram_feature_to_campaign 
Instance of Instagram_feature_to_campaign: Instagram Influencers.

CQ: When will the live script reading event take place? 
Answer: It will take place on February 15th 2022. 
Proposed terminology:  script_reading_event
Instance of  script_reading_event: February 15th 2022. 

CQ: What is the time for the script reading event? 
Answer: Doors will be open at 6:30pm and event will start at 7:00pm. 
Proposed terminology: time_script_reading_event
Instance of time_script_reading_event: 7:00pm.

CQ: What is the venue of the script reading event?
Answer: Script reading event will take place at Soulpepper Theatre,  50 Tank House Lane, Toronto, ON.
Proposed terminology: venue_event
Instance of venue_event: Soulpepper Theatre,  50 Tank House Lane, Toronto, ON.

CQ: Who can attend the script reading event? 
Answer: Ticket or Family Pass Holders can attend the event. 
Proposed terminology: attendee_event 
Instance of  attendee_event: Ticket or Family Pass Holders 

CQ: Who will be reading the script at the event? 
Answer: Actors and performers, who are associated with the show will be reading the script. 
Proposed terminology: readers_reading_script
Instance of  readers_reading_script: Actors and performers.

CQ: What we are offering to targeted audience on the days of performance? 
Answer: We are offering free parking at the venue. 
Proposed terminology: offer_days_of_performance
Instance of offer_days_of_performance: Free Parking 

CQ: What we are offering to targeted audiences post performance? 
Answer: We are offering micro mentorship opportunities to share experience and knowledge with the community members.
Proposed terminology: offer_post_event
Instance of offer_post_event: micro mentorship opportunities

CQ: Who can avail these micro mentorship opportunities
Answer: Any audience member between the age of 18-40 years old, who are interested to expand their knowledge on performance art. 
Proposed terminology: micro_mentorship_opportunities
Instance of micro_mentorship_opportunities: 18-40 years old, interested in performance art. 

CQ: Where these micro mentorship opportunities can take place? 
Answer: These opportunities will be designed as per the needs of the participants and availability of the mentors. Most of its part will be remote learning.  
Proposed terminology: location_micro_mentorship
Instance of location_micro_mentorship: Remote

CQ: How can the interested applicants connect with the creators? 
Answer: An application form is available on our website. Interested applicants can submit that form or email us at Offers@bhroductions.ca if case if they have any query.
Proposed terminology: contact_micro_mentorship
Instance of contact_micro_mentorship: Application form on the website or email at Offers@bhroductions.ca

# 4. Desired Outcome:
At Least 25-40% of the attendees are from this cultural groups.

# 5. Perspective:
Artist Creator

# 6. Stakeholder Group:
Audience members
Artists

# 6. Additional info:
_Here's more information._

**Comment by mariel-marshall** [comment]
@Gursimrandatla I wonder if this User Story can be modified to better fit your desired outcome which has to do with measuring whether your outreach efforts (reaching 25-40% from a specific cultural group) was effective. How would you go about measuring this? A proposal: 

Motivation: 
The connected User Story is about increasing awareness of your show to specific cultural groups. After taking actions to make these audiences aware, you now need to evaluate whether your outreach was effective.

**Need**
To determine whether attendees of the show fall into my target demographic cultural group (South Asian, Asian, Caribbean, and Asian-Pacific Islander audiences). 

**Action:** 
-I ask attendees at the event to answer an anonymous survey where they can self-identify their ethnicity.
-...other ideas on how you would determine whether your efforts were successful?

**Desired Outcome:**
At Least 25-40% of attendees are from (South Asian, Asian, Caribbean, and Asian-Pacific Islander audiences) cultural groups.

Does this make sense in terms of what you were hoping to achieve with the User Story?
